blog/2024-11-07-update-to-wit-package-management-in-wasmcloud/index.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: "Update to WIT package management in wasmCloud"
+authors: [ericgregory]
+image: './images/update-to-wit-package-management.webp'
+slug: 2024-11-08-update-to-wit-package-management-in-wasmcloud
+date: "2024-11-08"
+description: "How WIT package management in wasmCloud is evolving"
+---
+
+![TK](./images/update-to-wit-package-management.webp)
+
+wasmCloud's language-agnostic [interfaces](https://wasmcloud.com/docs/concepts/interfaces) are defined using **WebAssembly Interface Type (WIT) packages**, which can be **stored as OCI artifacts** in your registry of choice.
+
+As of `wash` v0.36, we're taking a big step forward in WIT package management, deprecating our usage of the `wit-deps` utility in favor of the OCI-compatible `wkg`. In this post, we'll take a look at the changes, break down exactly what's happening under the hood, and explain what it means for users.
+
+### What is changing?
+
+For some time now, `wash` didn’t have any knowledge of dependencies and instead expected the user to use a separate `wit-deps` utility to fetch and update WIT dependencies as raw `.wit` files in the `/wit/deps` directory of a wasmCloud project.
+
+That was originally intended as a brief, stop-gap solution, but `wit-deps` ended up serving as the stop-gap that just kept giving. As standards and approaches across the wider Wasm ecosystem were finalized, `wit-deps` fulfilled a critical role for a good deal longer than expected, and maintainer [Roman Volosatovs](https://github.com/rvolosatovs) in particular deserves a shout-out for the utility's above-and-beyond service.
+
+Now as of `wash` v0.36, `wit-deps` usage is being deprecated in favor of a new set of ecosystem tooling called [**Wasm Package Tools**](https://github.com/bytecodealliance/wasm-pkg-tools) and particularly the `wkg` (pronounced "wackage") command line tool.
+
+`wash` integrates `wkg` as a library, so there's no need to download a new tool. Whether standalone or as part of `wash`, `wkg` handles the following jobs:
```

Comment:
`wkg` is not the library, it's [`wasm-pkg-tools`](https://github.com/bytecodealliance/wasm-pkg-tools)

Reply:
Thanks, updated!